index.html

```diff
@@ -100,6 +100,8 @@ <h3>Scope</h3>
 			<section id="terminology">
 				<h3>Terminology</h3>
 
+				<p>This specification depends on the Infra Standard [[!infra]].</p>
+
```

Comment:
I am not sure it is appropriate to refer to the infra document normatively. I will have to check with the director, though, but there are only a few what wg documents (html, fetch, probably dom) that can be referred to normatively.

Reply:
Actually, I just got a note that it _is_ permitted to refer to infra normatively. I guess we can leave this as is, then.